dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyTreeFormatter.java

```diff
@@ -98,19 +98,23 @@ private static void formatDependencyPathTree(
   }
 
   /**
-   * Builds ListMultiMap that represents a Maven dependency tree of parent-children relationship.
-   * The root node is represented as an empty {@link DependencyPath} and its children are
-   * the values for the root node.
+   * Builds ListMultimap that represents a Maven dependency tree of parent-children relationship.
+   * Each node in the tree has a corresponding key in the ListMultimap and the children of the node
```

Comment:
Break into two sentences:

Each node in the tree has a corresponding key in the ListMultimap.
The value associated with that key is a list of the children of the node.

Reply:
Split the sentence.